### DISCUSSION TITLE
Help me, plis

### DISCUSSION THREAD

**Discussion by zeromiind** [JunkFood02/Seal#1401] [General]
So, good morning, I'm new to github and the world of programming, and I have some doubts that I would appreciate your answers.
I downloaded Seal on my cell phone, and I liked it too much, both the interface and the result. Then **I want to have it on my laptop but** I have seen that its for android and I do not know if its possible to have it on pc, and if so, **how do I get it or download** ?
**Thanks**

**Comment by rayanio** [comment]
Hi, Seal is only available for Android, which means there is no official PC version. There are plenty of other platforms on **Google** that offer the same thing as Seal, but if you really want to have it on your PC as an application, I would recommend you download an **Android emulator** such as Bluestacks or even Nox Player, and install it from there.

**Reply by zeromiind** [reply]
ALR, thanks. last question, after the installation, i can **delete the emulator**?

**Reply by rayanio** [reply]
No, you can't delete the emulator completely, otherwise it won't even launch.

**Reply by zeromiind** [reply]
oh thanks now.
